app/views/integrations/index.html.erb

```diff
@@ -29,6 +29,14 @@
       	<%= link_to "Create Slack Integration", new_slack_integration_path,class: 'btn btn-default btn-lg btn-block' %>
       </div>
     </div>
+    <div class="row">
```

Comment:
I think this would be a little clearer if this went below the list of slack integrations. When you have both slack and SQS integrations it's hard to tell what's going on. You might also consider playing with the spacing/maybe indenting the lists of integrations to make the hierarchy clearer.